lib/galaxy/datatypes/microarrays.py

```diff
@@ -1,6 +1,10 @@
 import logging
 
 from galaxy.datatypes import data
+from galaxy.datatypes.binary import (
```

Comment:
```suggestion
from galaxy.datatypes.binary import (  # noqa: F401
```

This is needed to silence the linter warnings.